newdle/client/src/components/NewdlesParticipating.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import flask from 'flask-urls.macro';
+import {Container, Icon, Label, Placeholder} from 'semantic-ui-react';
+import {useHistory} from 'react-router';
+import {serializeDate, toMoment} from '../util/date';
+import client from '../client';
+import {usePageTitle} from '../util/hooks';
+import styles from './MyNewdles.module.scss';
+
+export default function NewdlesParticipating() {
+  const [participations, loading] = client.useBackend(() => client.getNewdlesParticipating(), []);
+  usePageTitle("Newdles I'm participating in");
+
+  let content;
+  if (loading || participations === null) {
+    content = (
+      <>
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+        <Placeholder className={styles.newdle} />
+      </>
+    );
+  } else if (participations.length === 0) {
+    content = (
+      <div className={styles['no-newdle-container']}>
+        {/* eslint-disable-next-line jsx-a11y/accessible-emoji */}
+        <h2>You are not part of any newdles yet.</h2>
+      </div>
+    );
+  } else {
+    participations
+      .sort((a, b) => new Date(a.newdle.final_dt) - new Date(b.newdle.final_dt))
```

Comment:
I think some comments on each of the sorting lines could be helpful, since it'd be easier to read english instead of javascript to understand the sorting logic here ;)

Reply:
Agree.